docs/core/whats-new/whats-new-in-dotnet-core-2-0.md

```diff
@@ -0,0 +1,138 @@
+---
```

Comment:
I think we should follow the same pattern we've used for framework. A new folder requires an index.md file. So I'd make this the index.md file and just call it What's new in .NET Core, without version.